builtin/game/features.lua

```diff
@@ -3,6 +3,7 @@
 core.features = {
 	glasslike_framed = true,
 	nodebox_as_selectionbox = true,
+	independent_selectionbox = true,
```

Comment:
Should be moved to the bottom of the list since all other were added in a chronological order.